src/components/Button/Button.jsx

```diff
@@ -1,9 +1,11 @@
 import "./Button.scss";
 
-const Button = ({buttonText, handleClick}) =>{
+// we have designed all the styles for the project, you will need to call the classNaming for the specific style you need.
+
+const Button = ({buttonText, handleClick, classNaming}) =>{
```

Comment:
Is `classNaming` the best name for this prop? It is not a descriptive name specific to `button` - maybe `buttonClass` or `buttonStyle` would be better?